ci-operator/step-registry/ipi/conf/capability/additionalcaps/ipi-conf-capability-additionalcaps-commands.sh

```diff
@@ -104,6 +105,20 @@ while [[ -z "${selected_capability}" ]]; do
         echo "WARNING: MachineAPI is selected, but it requires on IPI, could not be disabled!"
         selected_capability=""
         ;;
+    "CloudControllerManager")
+        platform_type=$(oc get infrastructure cluster -ojsonpath='{.status.platformStatus.type}')
```

Comment:
This is config step running before cluster installation, we could not run `oc` command to get required field.
Maybe we can use `CLUSTER_TYPE` or read from install-config to get platform_type and ccm_type?